google-cloud-pub-sub-grpc/src/main/scala/org/apache/pekko/stream/connectors/googlecloud/pubsub/grpc/javadsl/GooglePubSub.scala

```diff
@@ -95,7 +98,10 @@ object GooglePubSub {
         Source
           .tick(Duration.ZERO, pollInterval, request)
           .mapAsync(1, client.pull(_))
-          .mapConcat(_.getReceivedMessagesList)
+          .mapConcat(
```

Comment:
Can we add a TODO/comment here that points to https://contributors.scala-lang.org/t/better-type-inference-for-scala-send-us-your-problematic-cases/2410/183 

Its likely that this will be fixed in a future version of Scala 3 at which point it would get backported to Scala 3.3.x series and we can remove this workaround.

Reply:
done